README.md

```diff
@@ -29,7 +29,21 @@ Install tkinter to be able to display graphs
 `sudo apt install python3-tk`
 
 ### Windows  
-Help wanted  
+Make sure pip for python 3 is installed  
+`pip3 --version` should not print an error  
+  
+Install pipenv via pip  
+`pip install pipenv`  
+
+Clone the repo and extract it
```

Comment:
If you do `git clone` for the repo, the repo is extracted for you. I assume you downloaded it like this  
![image](https://user-images.githubusercontent.com/8918762/95605900-3555a000-0a1f-11eb-895e-f5ee27ce71a8.png)
and that's why you had to extract it.  
Could you please make it so that the instructions to get the repo are either download and extract or git clone or both?

Reply:
Yeah. Git for Windows is needed to use git. I prefer using wget to grab the repo zip or simply directly download. I'll look into it.

Reply:
I'll just change it to only git clone so the `cd` after it is less prone to error